### ISSUE TITLE
List of All Orders

### ISSUE THREAD

**Issue by Masquerade0097** [fossasia/open-event-orga-app#1026]
### Feature Request

**Feature Description**

List all the Orders Related to an Event. There should be an Option in the Drawer to view List of all Orders for an Event.
   
**Parent Issue**

**Would you like to work on the issue?**

Yes

**Comment by Masquerade0097** [comment]
@iamareebjamal Should I proceed with this ?

**Comment by Masquerade0097** [comment]
There is no endpoint available for fetching the list of Orders for an Event. The avilable endpoint is only for listing all the Orders. 

Which endpoint do you think will be more useful for the app, fetching all orders for an Event or all orders for a particular Attendee?

@srv-twry has created a issue in Server regarding implementing Orders endpoint. Maybe I can help in implementing the particular endpoint.

**Comment by srv-twry** [comment]
@Masquerade0097 Yes, please create a child issue for it and then implement it :)

**Comment by iamareebjamal** [comment]
Both are important
